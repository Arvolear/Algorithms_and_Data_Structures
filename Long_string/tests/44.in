94980443480439216073102033500290680847
852