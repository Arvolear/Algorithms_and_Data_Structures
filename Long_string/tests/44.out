94980443480439216073102033500290681699
94980443480439216073102033500290679995
80923337845334212094282932542247660081644
111479393756384056423828677817242583